packages/ui-date-time-input/src/DateTimeInput/props.ts

```diff
@@ -28,7 +28,7 @@ import type { FormMessage } from '@instructure/ui-form-field'
 import type { InteractionType } from '@instructure/ui-react-utils'
 import { I18nPropTypes } from '@instructure/ui-i18n'
 import type { Moment } from '@instructure/ui-i18n'
-import PropTypes from 'prop-types'
+import PropTypes, { string } from 'prop-types'
```

Comment:
This made me confused first, can we just use PropTypes.string where it is used? I'd prefer that for consistency's stake, but I guess this works too :)

Reply:
done